tests/test_comments.py

```diff
@@ -180,8 +180,103 @@ class OAuthCommentTest(OAuthPRAWTest):
     def test_raise_invalidcomment_oauth(self):
         fullname = '{0}_{1}'.format(self.r.config.by_object[Comment],
                                     self.comment_deleted_id)
-        self.r.refresh_access_information(self.refresh_token['submit'])
```

Comment:
The cassette did not need to be changed however this needs to be reordered as it seems that previously a 'submit' access token was allowed on /api/info but now it requires the read scope and that alone.